cmake/compiler/gcc/compiler_flags.cmake

```diff
@@ -182,6 +182,9 @@ set_compiler_property(PROPERTY debug -g)
 # Flags to save temporary object files
 set_compiler_property(PROPERTY save_temps -save-temps=obj)
 
+# Flag to specify linker script
+set_compiler_property(PROPERTY linker_script -T)
```

Comment:
should go to the `linker/ld/gcc` front-end flags here:
https://github.com/zephyrproject-rtos/zephyr/blob/main/cmake/linker/ld/gcc/linker_flags.cmake